pkg/setting/setting.go

```diff
@@ -273,6 +273,7 @@ type Cfg struct {
 	PluginsAppsSkipVerifyTLS bool
 	PluginSettings           PluginSettings
 	PluginsAllowUnsigned     []string
+	MarketPlaceUrl           string
```

Comment:
[Go initialisms should have consistent case](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)
```suggestion
	MarketPlaceURL           string
```